GitPrompt.ps1

```diff
@@ -145,13 +145,31 @@ function Write-GitStatus($status) {
     }
 }
 
-if(!(Test-Path Variable:Global:VcsPromptStatuses)) {
-    $Global:VcsPromptStatuses = @()
+$VcsPromptStatusesVariableName = "VcsPromptStatuses"
+
+if (-not (Test-Path Variable:Global:$VcsPromptStatusesVariableName)) {
+    New-Variable -Name $VcsPromptStatusesVariableName -Value @() -Scope Global

```

Comment:
Out of curiosity, why `New-Variable ...` versus the simpler `$Global:VcsPromptStatuses = ...`?